bin/www

```diff
@@ -12,7 +12,7 @@ var http = require('http');
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || '8000');
+var port = normalizePort(process.env.PORT || '3000');
```

Comment:
That was redundant and was removed after the First PR itself, it is express standart to keep it in this file, app.js no longer has it.